### DISCUSSION TITLE
What is this library for?

### DISCUSSION THREAD

**Discussion by adhxtry** [encode/databases#354] [Q&A] [answered]
I mean I know this is for database like postgres, mysql, sqlite. But I m not sure how to use this library or is this library is what i m looking for. If there are more examples available then pls drop the like, or deep documentation. Thanks in advance :)

For now i have looked:
[databases](https://www.encode.io/databases/)

Also how to make connections, more info about the library.

**Comment by rafalp** [comment, marked as answer]
This library is asynchronous database access layer (DBAL) for Python. It gives you unified interface for connecting and querying different database systems in asynchronous fashion.

[Quickstart](https://github.com/encode/databases#quickstart) has example for connecting to SQLite database and running few queries against. There's also [connections and transactions](https://www.encode.io/databases/connections_and_transactions/) examples in the docs.

Can you share what's not obvious for you in the docs? They seem pretty self-explanatory looking at the feedback we've got so far.

**Reply by lovelydinosaur** [reply]
@rafalp 👉 Process tip... Please do feel free to go ahead and mark questions as answered.

Am still getting to grips with discussions and the slightly different UI, but I'm generally green-ticking stuff once I've responded to it with a good initial pass. 😀

<img width="149" alt="Screenshot 2021-06-15 at 14 01 37" src="https://user-images.githubusercontent.com/647359/122057148-388d0100-cde2-11eb-8f78-a6cc45e20bfd.png">

**Reply by rafalp** [reply]
Usually I give OP (OA?) day or two to reply to my answer before making question as answered myself, but if Encode's policy is to be proactive with those its fine with me ;)

**Reply by lovelydinosaur** [reply]
Fair enough - either way is good with me.
Personally I wouldn't *ever* manage to keep track of following up unless I just do it there & then. 🤣
